lib/tasks/usage_limits.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+namespace :usage_limits do
+  desc 'Deletes instances of UsageLimit for metrics of a backend, that is not linked to the product of the plan anymore'
```

Comment:
Do we add tests for rake tasks? If so, you could add a test, very similar to the one you already wrote.

Reply:
Some rake tasks do have tests, but definitely not all.
I certainly could have written one, but I was a bit lazy, given that this task is probably going to be a one-off, executed just one time. Not sure if it's a valid excuse though :grimacing:

Reply:
Fine, it's ok

Reply:
I've added a test in https://github.com/3scale/porta/pull/3996/commits/56ac846e34b30a747d9869a9cec374a21b8d31a1